kview/src/main/resources/dev/ikm/komet/kview/controls/public-id-control.css

```diff
@@ -3,10 +3,9 @@
        where the height is not changed as the mouse cursor enters and exits
        the control.  30px seems to work well for the current settings for
        fontSize and the icon scale. */
-    -public-id-height: 30px;
 
-    -fx-pref-height: -public-id-height;
-    -fx-min-height: -public-id-height;
+    -fx-pref-height: 22px;
+    -fx-min-height: 22px;
```

Comment:
can we use the variable like we were using. That is: 

    -public-id-height: 22px;

    -fx-pref-height: -public-id-height;
    -fx-min-height: -public-id-height;